ISICbalance/ViewModels/BalanceViewModel.swift

```diff
@@ -28,17 +27,14 @@ extension BalanceViewModelingActions where Self: BalanceViewModeling {
 
 final class BalanceViewModel: BaseViewModel, BalanceViewModeling, BalanceViewModelingActions {
     typealias Dependencies = HasRequestManager
-    private let dependencies: Dependencies // TODO: není potřeba
 
-    lazy var balance = Property<String>(initial: "0 Kč", then: getBalanceAction.values.map { $0.balance }) // TODO: kč lokalizovat a přilepovat až ve view a vůbec.. možná spíš vracet číslo než string
+    lazy var balance = Property<String>(initial: "0", then: getBalanceAction.values.map { $0.balance }) // TODO: možná spíš vracet číslo než string -> Opravdu to mám převádět pomocí formatteru do double a pak to zase formátovat do double s čárkou?
```

Comment:
bylo by fajn to mít jako číslo 😊 ale není asi moc potřeba. Nepracuješ s tím nikde dál. Pokud to doděláš, tak bych se zamyslel, jestli je potřeba to zobrazovat jako `Double`, tedy s desetinnou čárkou, nebo jako `Int`.

Reply:
Asi bude nejlepší to oříznout a prostě dávat stav v účtu v celých korunách. Ty halíře asi nedávají moc smysl.